board/views.py

```diff
@@ -1,36 +1,82 @@
-                 '../../static/board/img/screen3.png',
-                 '../../static/board/img/screen1.png', '../../static/board/img/screen2.jpg',
-                 '../../static/board/img/screen3.png']
-    genres = ['Arcade', 'Action']
-    platforms = ['PC', 'PS4']
-    rating = {'users': [7.32, 123], 'critics': [7.11, 123]}
-    id = 1
+    def __init__(self, game_id):
+        params = {
+            'fields': 'id, cover.image_id, name, '
+                      'summary, screenshots.image_id, '
+                      'genres.name, release_dates, '
+                      'platforms.name, '
+                      'aggregated_rating, aggregated_rating_count, '
+                      'rating, rating_count ',
+            'filter[id][eq]': game_id
+        }
+
+        res = requests.post(igdb_url + 'games/', headers=igdb_headers, params=params).json()[0]
+
+        self.id = game_id
+        self.name = res['name']
+        self.desc = ' '.join([genre['name'] for genre in res['genres']])
+        self.desc_full = res['summary']
+        if 'cover' in res:
+            self.img_url = get_img_url(res['cover']['image_id'])
+        else:
+            self.img_url = get_img_url(res['screenshots'][0]['image_id'])
+        self.release = datetime.datetime.fromtimestamp(res['release_dates'][0])
+        self.screen_url = [get_img_url(item['image_id']) for item in res['screenshots']]
+        self.genres = [genre['name'] for genre in res['genres']]
+        self.platforms = [platform['name'] for platform in res['platforms']]
+        if 'rating' in res:
+            self.rating = [res['rating'], res['rating_count']]
+        else:
+            self.rating = ['', 0]
+        if 'aggregated_rating' in res:
+            self.aggregated_rating = [res['aggregated_rating'], res['aggregated_rating_count']]
+        else:
+            self.aggregated_rating = ['', 0]
+
     tweets = [['tweet_name', 'Pac-Man is a maze arcade game developed and released by Namco in 1980.'
                 'The original Japanese title of Puck Man was changed to Pac-Man for international releases as '
                 'a preventative',
                datetime.datetime.now()]] * 5
 
 
+class Filter:
+    def __init__(self):
+        params = {
+            'fields': 'name'
+        }
+
+        res_genres = requests.post(igdb_url + 'genres/', headers=igdb_headers, params=params).json()
+        res_platforms = requests.post(igdb_url + 'platforms/', headers=igdb_headers, params=params).json()
+
+        self.genres = [item['name'] for item in res_genres]
+        self.platforms = [item['name'] for item in res_platforms]
+
+
+
 def main(request):
-    games = [Game()] * 31
+    params = {
+        'filter[screenshots][not_eq]': 'null',
+        'filter[genres][not_eq]': 'null'
+    }
+    res = requests.post(igdb_url + 'games/', headers=igdb_headers, params=params).json()
+    games = []
+    for game in res:
+        games.append(Game(game['id']))
```

Comment:
I guess this results in calling igdb api for each game, right? Do you need all this data for just a list of games?